autoscheduler/frontend/src/components/SchedulingPage/Schedule/Schedule.tsx

```diff
@@ -372,7 +373,7 @@ const Schedule: React.FC = () => {
       );
     }
     return [DayOfWeek.MON, DayOfWeek.TUE, DayOfWeek.WED, DayOfWeek.THU, DayOfWeek.FRI].map(
-      (idx) => getMeetingsForDay(idx).map((mtg) => renderMeeting(mtg)),
+      (idx) => filterDuplicateMeetings(getMeetingsForDay(idx)).map((mtg) => renderMeeting(mtg)),
     );
```

Comment:
I'm fine with this implementation but I feel like moving all this logic to its own function in a helper file (like `meetingsForSection` in `meetingsForSection.ts`) would make this a bit cleaner, since in general it's a good idea to separate business logic from the components themselves